### DISCUSSION TITLE
A way to trigger alerts migrations manually again

### DISCUSSION THREAD

**Discussion by tonnyavery** [grafana/grafana#55220] [Alerting]
**What happened**: Alerts migration is run only on Grafana instance start. That can be too early for cases when dashboards are provisioned via API calls.

**What you expected to happen**: It would be great to have a way to manually trigger migrations again, so that new alerts get populated and it is possible to reconfigure API provisioning to work with the new alert system.

**How to reproduce it (as minimally and precisely as possible)**: Having old grafana dashboards with some alerts, start new Grafana 9.x and then use API calls to create a dashboard using existing JSON from the old system.

**Anything else we need to know?**:

**Environment**:
- Grafana version: 9.1.4 (2186d0bbeb)
- Data source type & version:
- OS Grafana is installed on: Linux
- User OS & Browser:
- Grafana plugins:
- Others:

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
